libs/coin-polkadot/src/logic/utils.test.ts

```diff
@@ -68,11 +43,11 @@ describe("canUnbond", () => {
           })),
         ],
       },
-    };
-    expect(canUnbond(account as PolkadotAccount)).toBeTruthy();
+    });
+    expect(canUnbond(account)).toBeTruthy();
   });
   test("can't unbond because unlockings is too much", () => {
-    const account: Partial<PolkadotAccount> = {
+    const account = createFixtureAccount({
```

Comment:
We already discussed this, unit tests have by definition no dependencies, we are a adding a small one

To me this is the role of integration tests to provide stubs/global fixtures and test features

I do not agree with this but we already spent too much energy for no reason as the team did not decide yet on a consensus and we are only two people arguing :zipper_mouth_face: 

https://softwareengineering.stackexchange.com/questions/287613/javascript-unit-testing-mocks-or-fixtures
https://softwareengineering.stackexchange.com/a/227985
https://softwareengineering.stackexchange.com/a/198456
https://softwareengineering.stackexchange.com/a/154255
https://paytonrules.com/post/2016-04-18-dont-use-fixtures/
https://medium.com/worldsensing-techblog/tips-and-tricks-for-unit-tests-b35af5ba79b1